site/docs-md/getting-started/index.md

```diff
@@ -5,19 +5,20 @@ There are two ways to start using Capacitor: adding Capacitor to an existing fro
 Capacitor provides a native mobile runtime and API layer for web apps. It does _not_ come with any specific
 set of UI controls which you will most likely need unless you're building a game or something similar.
 
-We strongly recommend starting a Capacitor project with your mobile frontend framework of choice (such as Ionic),
-though we also provide a blank starter for apps that aren't using a frontend UI framework, and a starter that uses Ionic and
-is ready for building a production-ready native app and Progressive Web App.
+We strongly recommend starting a Capacitor project with your mobile frontend framework of choice (such as [Ionic](https://ionicframework.com)).
 
 ## Before you start
 
 Make sure you have all the required [Dependencies](./dependencies) installed for the platforms you will be building for. Most importantly,
 make sure you update CocoaPods using `pod repo update` before starting a new project, if you plan on building for iOS using a Mac.
 
+## Adding Capacitor to an existing Ionic App
+
+[See here.](/docs/getting-started/with-ionic)
 
 ## Adding Capacitor to an existing web app
 
-Capacitor was designed to drop-in to any existing modern JS web app.
+Capacitor was designed to drop-in to any existing modern JS web app. Only a valid `package.json` file is required to get started.
```

Comment:
Maybe also mention that a folder with the web assets is needed? There was an issue about an user using Elm and putting all the files on the root instead of in a folder and that was causing out of memory because of the folder copying itself inside the platforms folders. The infinite copy is now fixed, but might be good to add that web assets must be in a folder (www as default). Also that you can use `--web-dir` option in init command to set it to a different folder.